csrankings-s.csv

```diff
@@ -1837,6 +1837,7 @@ Stephen James Wilson,University of Queensland,https://researchers.uq.edu.au/rese
 Stephen John Maybank,Birkbeck University of London,https://www.dcs.bbk.ac.uk/about/people/academic-staff/sjmaybank,gpyHJmcAAAAJ
 Stephen Kell,King's College London,https://www.humprog.org/~stephen/,4GHlcNoAAAAJ
 Stephen Lee,University of Pittsburgh,http://www.sci.pitt.edu/faculty-and-research/faculty-directory/stephen-lee,qsjlH3AAAAAJ
+Stephen M. Blackburn,Australian National University,http://users.cecs.anu.edu.au/~steveb,HgSTO7oAAAAJ

```

Comment:
Not full-time (>= 75%), in industry, no longer eligible for inclusion.

Reply:
In the interest of expedience, I'll manually remove his entry after merging this.